egs2/vggsound/cls1/db.sh

```
@@ -0,0 +1,388 @@
+SNIPS= # smart-light-en-closed-field data path
+SPGISPEECH=
+SPEECH_PROMPT_v2=
+STOP=
+SWBD=
+FISHER_CALLHOME_SPANISH=
+SWBD_NXT=
+THCHS30=downloads
+TIMIT=
+TSUKUYOMI=downloads
+VOXFORGE=downloads
+VGGSOUND=
```

Shellcheck (0.11.0)

[warning] VGGSOUND appears unused. Verify use (or export if used externally).

(SC2034)